clients/admin-ui/src/pages/login.tsx

```diff
@@ -164,6 +164,7 @@ const Login: NextPage = () => {
                             Username
                           </FormLabel>
                           <Input
+                            autoFocus
```

Comment:
Set focus to the Username input field when rendered and turned off autocomplete for both Username and Password input fields.